hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -428,12 +409,12 @@ void setHostnameResolver(final HostnameResolver hostnameResolver) {
         this.hostnameResolver = hostnameResolver;
     }
 
-    void setRabbitTemplate(final RabbitTemplate rabbitTemplate) {
-        this.rabbitTemplate = rabbitTemplate;
+    void setMessageConverter(final MessageConverter messageConverter) {
```

Comment:
Can this setter be removed? The `MessageConverter` is already given in the constructor and could be final I guess then.